test/unit/rules/no-invalid-link-title-test.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'no-invalid-link-title',
+
+  config: true,
+
+  good: [
+    '<a href="https://myurl.com">Click here to read more about this amazing adventure</a>',
+    '{{#link-to}} click here to read more about our company{{/link-to}}',
+    '<LinkTo>Read more about ways semantic HTML can make your code more accessible.</LinkTo>',
+    '<LinkTo>{{foo}} more</LinkTo>',
+    '<a href="https://myurl.com" title="New to Ember? Read the full tutorial for the best experience">Read the Tutorial</a>',
+  ],
+
+  bad: [
```

Comment:
The `@ember/routing/link-component` does support setting `title` (it uses [attributeBindings here](https://github.com/emberjs/ember.js/blob/master/packages/@ember/-internals/glimmer/lib/components/link-to.ts#L398)), so we need to account for it as both an argument and attribute.

Please add a bad example for each of the following snippets:

```hbs
{{#link-to 'some.route' title="Do the things"}}Do the things{{/link-to}}
```

```hbs
<LinkTo @route="some.route" @title="Do the things">Do the things</LinkTo>
```